components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FxaAccountManager.kt

```diff
@@ -62,6 +62,14 @@ internal sealed class Event {
     object FailedToFetchProfile : Event()
 
     object Logout : Event()
+
+    data class Pair(val pairingUrl: String) : Event() {
```

Comment:
@grigoryk could you confirm if this name is okay? Hopefully fits the rest of the state machine?

Reply:
fwiw, I think the name fits in :)

Reply:
:+1: